src/controller/user/controller.go

```diff
@@ -167,6 +173,22 @@ func (c *controller) Count(ctx context.Context, query *q.Query) (int64, error) {
 }
 
 func (c *controller) Delete(ctx context.Context, id int) error {
+	// cleanup project member with the user
+	if err := c.memberMgr.DeleteMemberByUserID(ctx, id); err != nil {
+		return err
+	}
+
+	// delete oidc metadata under the user
+	a, err := config.AuthMode(ctx)
```

Comment:
When the func in controller is called, he authmode is very likely populated to the ctx already:
https://github.com/goharbor/harbor/blob/60478f4990975ffc878f0b186843e7c0f5f87c52/src/server/middleware/security/security.go#L54

Reply:
Updated